orchagent/vnetorch.cpp

```diff
@@ -1857,6 +1859,13 @@ void VNetRouteOrch::createBfdSession(const string& vnet, const NextHopKey& endpo
         // when the device goes into TSA.  The following parameter ensures that these session are
         // brought down while transitioning to TSA and brought back up when transitioning to TSB.
         data.emplace_back("shutdown_bfd_during_tsa", "true");
+        if (has_monitor_interval)
+        {
+            FieldValueTuple fvTuple1("tx_interval", to_string(tx_monitor_interval));
```

Comment:
Please include sonic-mgmt tests with Rx/Tx timer to check if these are supported by vendor SAI for BFD hardware offload.

Reply:
ill add those tests after I have made the changes for directly connected nexthops.